src/module/Entity/src/Entity/Entity/Entity.php

```diff
@@ -253,6 +253,11 @@ public function isUnrevised()
         return $this->countUnrevised() > 0;
     }
 
+    public function isDead()
+    {
+        return !$this->getHead();
```

Comment:
Oh yeah, I guess you're right. But that covers too much, doesn't it? This would also cover new entities that have a revision that isn't approved yet.